color/map_palette_assignments.asm

```diff
@@ -0,0 +1,196 @@
+; $60 bytes for each tileset. Each byte is the palette number for a tile.
+; Remaining $a0 tiles aren't part of the tileset and are set to zero.
+; Refer to MapPaletteSets for clarification on specific colors
+; Each row of entries corresponds to a row of tiles on the tileset image
+MapPaletteAssignments:
+; OVERWORLD
+	db GRAY,   BROWN,  BROWN,  RED,    BROWN,  ROOF,   ROOF,   ROOF,   ROOF,   ROOF,   YELLOW, YELLOW, YELLOW, GRAY,   BROWN,  GRAY
+	db GRAY,   BROWN,  ROOF,   BROWN,  BLUE,   ROOF,   ROOF,   ROOF,   ROOF,   ROOF,   GRAY,   BROWN,  BROWN,  GREEN,  BROWN,  GRAY
+	db GRAY,   GRAY,   GRAY,   GRAY,   BROWN,  ROOF,   ROOF,   BROWN,  ROOF,   ROOF,   GRAY,   GRAY,   GREEN,  GREEN,  GREEN,  GREEN
+	db GREEN,  GRAY,   BROWN,  BROWN,  BROWN,  BROWN,  BROWN,  BROWN,  ROOF,   GRAY,   GRAY,   GRAY,   BROWN,  GREEN,  GREEN,  GRAY
+	db GREEN,  GREEN,  GRAY,   GRAY,   GRAY,   GRAY,   GRAY,   GRAY,   BROWN,  BROWN,  GRAY,   BROWN,  ROOF,   ROOF,   GRAY,   GRAY
+	db GREEN,  GREEN,  GREEN,  ROOF,   BROWN,  BROWN,  GRAY,   GRAY,   BROWN,  BROWN,  ROOF,   BROWN,  ROOF,   ROOF,   GRAY,   GRAY
+
+; REDS_HOUSE_1
```

Comment:
Why are these hardcoded colors? Don't they depend on the map palette set? For example, if I change the first byte of the REDS_HOUSE_1 map palette set from OUTDOOR_GREY to FOREST_TREES, GRAY actually becomes a non-gray color in Red's house. Am I right?